src/zcl_abapgit_repo_online.clas.abap

```diff
@@ -229,34 +224,36 @@ CLASS ZCL_ABAPGIT_REPO_ONLINE IMPLEMENTATION.
   ENDMETHOD.
 
 
-  METHOD initialize.
-    IF mv_initialized = abap_false.
-      refresh( ).
-    ENDIF.
+  METHOD has_remote_source.
+    rv_yes = abap_true.
   ENDMETHOD.
 
 
-  METHOD rebuild_local_checksums. "REMOTE
+  METHOD rebuild_local_checksums.
+
+    " TODO: method unify to base class !
 
-    DATA: lt_remote    TYPE zif_abapgit_definitions=>ty_files_tt,
+    DATA:
+          lt_remote    TYPE zif_abapgit_definitions=>ty_files_tt,
           lt_local     TYPE zif_abapgit_definitions=>ty_files_item_tt,
           ls_last_item TYPE zif_abapgit_definitions=>ty_item,
           lt_checksums TYPE zif_abapgit_persistence=>ty_local_checksum_tt.
 
-    FIELD-SYMBOLS: <ls_checksum> LIKE LINE OF lt_checksums,
+    FIELD-SYMBOLS:
+                   <ls_checksum> LIKE LINE OF lt_checksums,
```

Comment:
the DATA and FIELD-SYMBOLS in this method looks identical to the previous version, but there is a diff

Reply:
see `rebuild` comment above